hcl/hcl.go

```diff
@@ -99,9 +106,10 @@ func Parse(path string) (HCL, error) {
 // BuildRunners steps through the HCLConfig structs and maps each runner config type to the corresponding New<Runner> function.
 // All custom runners are reduced into a linear slice of runners and served back up to the product.
 // No runners are returned if any config is invalid.
-func BuildRunners[T Blocks](config T, tmpDir string, c *client.APIClient, since, until time.Time) ([]runner.Runner, error) {
+func BuildRunners[T Blocks](config T, tmpDir string, c *client.APIClient, since, until time.Time, redactions []*redact.Redact) ([]runner.Runner, error) {
```

Comment:
Not in scope for this PR but rather an observation that BuildRunners is still very complex. Despite my best effort to improve it last month we should probably get rid of the union type param and have separate functions for Host (later Platform) and Product - the repetition is still there with the type switch.